toor.js

```diff
@@ -53,6 +54,7 @@ if (cluster.isWorker) {
   const createServerShimmer = (original) => {
     return function shimCreateServer (...args) {
       const server = original.apply(this, args)
+      if (server.hasOwnProperty('httpAllowHalfOpen')) addShutdown(server)
```

Comment:
I'm using this shim for both `http` and `net`, so this is my lazy way of seeing which one I'm dealing with, and only applying it to `http`. If you're trying to do this with `net`, you've got to do other things beyond the scope of `toor` to make sure your sockets are cleared, so I thought it best to ignore `net` for now.